subworkflows/local/binning.nf

```diff
@@ -159,11 +158,8 @@ workflow BINNING {
     ch_versions = ch_versions.mix(MAG_DEPTHS_SUMMARY.out.versions)
 
     // Group final binned contigs per sample for final output
-    ch_binning_results_gunzipped_final = ch_binning_results_gunzipped
-        .groupTuple(by: 0)
-
-    ch_binning_results_gzipped_final = METABAT2_METABAT2.out.fasta.mix(MAXBIN2.out.binned_fastas)
-        .groupTuple(by: 0)
+    ch_binning_results_gunzipped_final = ch_binning_results_gunzipped.groupTuple(by: 0)
+    ch_binning_results_gzipped_final   = ch_binning_results_gzipped_final.groupTuple(by: 0)
```

Comment:
Hm input & output same name might be error prone??

Reply:
Good question... it's done for software versions all the time 
```
ch_versions = ch_versions.mix(<MY_PROCESS>.out.versions.first())
```
so I assumed it's ok, but might be a different case

Reply:
Hmyes, `groupTuple` and `mix` are quite different. I am just not sure.